src/@types/analytics-handler/index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * This is an interface for the analytics handler in petabox
+ * because we don't currently have a declarations file for it.
+ */
+export interface AnalyticsHandlerInterface {
```

Comment:
One of the cool things you can do with TypeScript is define the interface for javascript classes that weren't written in TypeScript. It lets you interact with it as though it were native TypeScript and your IDE/TypeScript compiler will ensure that you are using it correctly.

This here is the interface to `analytics.js` from petabox so we can inject that into the `donation-form` so it can do its analytics.

Ideally, I'd like to pull `analytics.js` out of petabox and into its own module and this file would live with it there so any of our consumers (other npm modules and petabox itself) could pull it in.

--

There's an organization called [DefinitelyTyped](https://definitelytyped.org) that has been building/collecting types for tons of plain javascript libraries so you can use them in TypeScript. We're using some of them in here... all of the `@types/<foo>` dependencies in `package.json`